doc/source/ray-dashboard.rst

```diff
@@ -0,0 +1,357 @@
+
+- Created and killed actors
+- Actor stats such as actor status, number of executed tasks, pending tasks, and memory usage.
+- Actor hierarchy and dependency.
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/dashboard/Logical-view-basic.png
+    :align: center
+
+Ray Config
+~~~~~~~~~~
+You can see the autoscaler configuration at the Ray Config tab. Look at the picture to see what it provides.
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/dashboard/Ray-config-basic.png
+    :align: center
+
+Tune
+~~~~
+Go to the Tune tab to see running Tune jobs and see their statistics in details. It offers
+
+- Tune jobs and their status
+- Hyperparameters each job is experimenting on.
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/dashboard/Tune-basic.png
+    :align: center
+
+Advanced Usage
+--------------
+
+Killing Actors
+~~~~~~~~~~~~~~
+You can kill actors when some actors are hanging or not in progress through the dashboard URL.
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/dashboard/kill-actors.png
+    :align: center
+
+Debugging a Blocked Actor
+~~~~~~~~~~~~~~~~~~~~~~~~~
+You can find hanging actors through the Logical View tab.
+
+If creating an actor requires resources (e.g., CPUs, GPUs, or other custom resources) that are not currently available, the actor cannot be created until those resources are added to the cluster or become available.
+This can cause an application to hang. To alert you to this issue, infeasible tasks are shown in red in the dashboard, and pending tasks are shown in yellow.
+You can find this information from the *Logical View* tab in the dashboard.
+
+.. code-block:: python
+
+  import ray
+  
+  ray.init(num_gpus=2)
+
+  @ray.remote(num_gpus=1)
+  class Actor1:
+      def __init__(self):
+          pass
+
+  @ray.remote(num_gpus=4)
+  class Actor2:
+      def __init__(self):
+          pass
+
+  actor1_list = [Actor1.remote() for _ in range(4)]
+  actor2 = Actor2.remote()
+
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/dashboard/dashboard-pending-infeasible-actors.png
+    :align: center
+
+This cluster has two GPUs, and so it only has room to create two copies of Actor1. As a result, the rest of Actor1 will be pending.
+
+You can also see it is infeasible to create Actor2 because it requires 4 GPUs which is bigger than the total gpus available in this cluster (2 GPUs). 
+
+Inspect Memory Usage
+~~~~~~~~~~~~~~~~~~~~
+You can detect local memory anomalies through the Logical View tab.
+
+.. image:: https://raw.githubusercontent.com/ray-project/images/master/docs/dashboard/dashboard-inspect-local-memory-usage.png
+    :align: center
+
+Let's see how to use this information.
+
+if NumObjectIdsInScope, NumLocalObjects, or UsedLocalObjectMemory keeps growing without bound, it can lead to OOM errors or eviction of objectIDs that your program still wants to use. 
```

Comment:
Maybe remove L134?

Reply:
For this entire Inspect Memory Usage section, I would just say:

```
You can detect local memory anomalies through the Logical View tab. if NumObjectIdsInScope, NumLocalObjects, or UsedLocalObjectMemory keeps growing without bound, it can lead to OOM errors or eviction of objectIDs that your program still wants to use. 

.. image..
```